### DISCUSSION TITLE
Idea: Integrate ENVST's campus map features into a MapBox view

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5541] [Ideas]
I don't know where this would live in the app, but it should be possible to take the custom layers from a Google My Map and render them as features on a MapBox map. 

> It is my pleasure to share with you the collaborative work product of two groups of EnvSt 237 students and an Environmental Studies Program Fellow.  They generated multiple iterations of campus sustainability maps using a few different platforms, and landed on using Google My Maps as one yielding low barriers of entry to users as well as to those maintaining the map context.  We deeply appreciate the assistance of colleagues across campus, especially our friends in Facilities.  Enjoy exploring the map, and we hope you find it useful in engaging students, colleagues, alumni, and visitors.
>
> You can access the map by clicking the link
Campus Sustainability Map (https://drive.google.com/open?id=1lKNYHPLmSOAxJI-gyXWJFhywX9D0q23O)

**Comment by rye** [comment]
Would we consider this part of the "better campus map" story?

**Comment by hawkrives** [comment]
Yep!
